### PR TITLE
Changed Homepage buttons

### PR DESCRIPTION
**Pull request by pateldhruv0248**
1. Changed positions of 'Give away'  and  'Request'  rounded buttons.
- Added static string '20 donations made till now :)' at the bottom of the page.

2. rounded_button.dart file changed :-

- added 2 new variables 'height' and 'width'
- shape of buttons changed to circle using ClipOval()
- text wrapping applied

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -98,7 +98,7 @@ packages:
       name: crypto
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.1.5"
```
Comment (iamprayush) [inline pubspec.lock:101]: Revert this change.

**Review by iamprayush — CHANGES_REQUESTED** [review]
Problems:
- Long name overflows
- Donations made till now text is too small
- Add "FFG" in the topbar for the homescreen.

File: lib/components/circular_button.dart [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  final width;
+
+  CircularButton({
+    this.colour,
+    this.title,
+    this.pressed,
+    this.splashColour = Colors.deepOrange,
+    this.height = 150.0,
+    this.width = 150.0
+  });
```
Comment (iamprayush) [inline lib/components/circular_button.dart]: Make sure the order is maintained. (`splashColour` should be after `pressed`).

File: lib/components/circular_button.dart [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    return ClipOval(
+      child: Material(
+        color: kPrimaryColor, // button color
+        child: InkWell(
+          splashColor: splashColour,
```
Comment (iamprayush) [inline lib/components/circular_button.dart]: Use `this.` here and elsewhere for all member variables.

File: lib/components/circular_button.dart [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import 'package:FoodForGood/constants.dart';
+import 'package:flutter/material.dart';
+
+class CircularButton extends StatelessWidget {
+  final Color splashColour;
+  final Color colour;
+  final String title;
+  final Function pressed;
+  final height;
+  final width;
```
Comment (iamprayush) [inline lib/components/circular_button.dart]: Mention the data type.

**Review by iamprayush — CHANGES_REQUESTED** [review]
1. "FFG" in the topbar should have the background color.
2. Add the "What would you like to do today?" line as shown in this mock:
Screenshot from 2020-09-19 15-43-52
3. Number of donations made line should be bigger and the number should be highlighted (bold, bigger and maybe different color).

**Comment by pateldhruv0248** [comment]
Done

File: lib/components/circular_button.dart (hunk 2 above)
Comment (pateldhruv0248) [reply lib/components/circular_button.dart]: Done

File: lib/components/circular_button.dart (hunk 3 above)
Comment (pateldhruv0248) [reply lib/components/circular_button.dart]: Done

File: lib/components/circular_button.dart (hunk 4 above)
Comment (pateldhruv0248) [reply lib/components/circular_button.dart]: Done.

**Review by iamprayush — APPROVED** [review]
LCTM! (Looks ChakaChak to me!)
